### ISSUE TITLE
função escreva com "\n" embutido

### ISSUE THREAD

**Issue by barichello** [UNIVALI-LITE/Portugol-Studio#841]
Seria muito bom ter uma função como a escreva com um "\n" embutido no final. Algo como uma "escrevanl".
Sempre acho quase constrangedor ter que explicar o "\n" para um iniciante.

**Comment by noschang** [comment]
Boa tarde @barichello. Obrigado pela sua sugestão. Já marcamos essa issue como pedido de uma nova feature e vamos deixar em aberto por enquanto. Se houverem mais pedidos por esse funcionalidade podemos avaliar a melhor forma de implementar.

**Comment by noschang** [comment]
O que vocês acham @AdsonEsteves e @elieserdejesus? Não deve ser difícil de implementar a funcionalidade, o problema maior será a a parte do analisador semântico. Talvez uma nova função não seja a solução ideal também.

O que vocês acham de a gente dar suporte a string multi-linha? Podemos restringir o uso de string multi-linha para uso em variáveis apenas. Por exemplo:

```java
// Isso funciona
cadeia texto = "Esse é um
texto com 
múltiplas linhas
"

escreva(texto)

//---------------------------------------------------//

// Isso nem compila
escreva("Esse é outro texto
com múltiplas
linhas
")
```

**Comment by AdsonEsteves** [comment]
@noschang 
O escrevaln me parece mais interessante;
string multilinha me parece estranho lidar ela diferentemente nos dois casos. Me parece ter mais chance de confundir o usuário e isso não é comum nas linguagens.

**Comment by noschang** [comment]
@AdsonEsteves, certo. Acho que dá pra aproveitar bastante coisa do analisador semântico
